commands/get-unix-timestamp/command.toml

```diff
@@ -0,0 +1,16 @@
+name = "Get Current UNIX Timestamp"
```

Comment:
Maybe "Copy Current Unix Timestamp" (this will be precise if the command immediately copied the timestamp) or "Show Current Unix Timestamp" (this likely works better for the current command's design where you show a single option to copy the timestamp)?